README.md

```diff
@@ -117,25 +174,27 @@ final CMAArray<CMASpace> array =
 });
 ```
 
-Please note that the default `CMACallback` has an empty `onFailure()` implementation, which makes it optional to override it.
+> Note: The default `CMACallback` has an empty `onFailure()` implementation. If failures are of interest, overriding this methods is immanent.
```

Comment:
It should either be -overriding this method or overriding these methods 
Also, the word 'immanent' can be 'mandatory'